priv/templates/install/beacon_data_source.ex

```diff
@@ -0,0 +1,9 @@
+defmodule <%= inspect beacon_data_source.module_name %> do
+  @behaviour Beacon.DataSource.Behaviour
+
+  def live_data("<%= beacon_site %>", ["home"], _params), do: %{vals: ["first", "second", "third"]}
+
+  def live_data("<%= beacon_site %>", ["blog", blog_slug], _params), do: %{blog_slug_uppercase: String.upcase(blog_slug)}
```

Comment:
```suggestion
  def live_data("<%= beacon_site %>", ["blog", blog_slug], _params) do
    %{blog_slug_uppercase: String.upcase(blog_slug)}
  end
```

So the generated file is left formatted.